docs/writing-tests/addon-vitest.mdx

```diff
@@ -0,0 +1,346 @@
+      tags: {
+        include: ['test', 'spec'],
+        exclude: ['docs-only'],
+      },
+    }),
+  ],
+}
+```
+
+If the same tag is in both the `include` and `exclude` arrays, the `exclude` behavior takes precedence.
+
+## FAQ
+
+### How to ensure my tests can find assets in the public directory?
+
+If your stories use assets in the public directory and you're not using the default public directory location (`public`), you need to adjust the Vitest configuration to include the public directory. You can do this by providing the `publicDir` option in the Vitest configuration file.
+
+```ts
+TK
+```
+
+### How to debug my tests in Storybook?
+
+The plugin will attempt to provide links to the story in Storybook when tests fail, for [debugging](#debugging) purposes.
+
+If the URLs are not working when running tests in watch mode, you should check two configuration options:
+
+- [`storybookUrl`](#storybookurl): Ensure this URL is correct and accessible. For example, the default is `http://localhost:6006`, which may not use the same port number you're using.
+- [`storybookScript`](#storybookscript): Ensure this script is correctly starting Storybook.
+
+If the URLs are not working when running tests in CI, you should ensure the Storybook is built and published before running the tests. You can then provide the URL to the published Storybook using the `storybookUrl` option. See the [In CI](#in-ci) section for an example.
+
+### How to apply custom Vite configuration?
+
+If you have custom operations defined in [`viteFinal`](../api/main-config/main-config-vite-final.mdx) in your `.storybook/main.js|ts` file, you will need to translate those into the Vitest configuration. This is because the plugin does not use the Storybook Vite configuration.
+
+```ts
+TK
+```
+
+### Why do we recommend browser mode?
+
+```
+1. It’s a real browser environment. JSDom/HappyDom are simulations with shortcomings.
+2. https://vitest.dev/guide/browser/#motivation
+```
+
+### How to use WebDriver instead of Playwright?
+
+```
+https://vitest.dev/config/#browser-provider
+```
+
+### How to use a browser other than Chromium
+
+```
+https://vitest.dev/config/#browser-46-name
+```
+
+### How is this different from the test runner?
+
+```
+1. TR requires an SB instance to be running; this does not (except for debugging)
+2. TR runs SB and listens to results; this transforms stories (using portable stories) into tests
+3. TR is based on Jest; this is based on Vitest
+4. This is more configurable and more simple than TR
+    1. TR is always a separate command; this is just Vitest (`yarn test`)
+5. This is faster than TR
+    1. Needs benchmarks
+        1. a sandbox run in our monorepo
+            1. Vitest plugin: 1m 6s
+            2. Test-runner: 1m 14s + SB build & publish time
+```
+
+### Why does the `add` command stop in some cases?
+
+TK
+
+## API
+
+### Exports
+
+`@storybook/experimental-addon-vitest/plugin`
+
+TK
+
+### Options
+
+#### `configDir`
```

Comment:
If people have non standard locations for .storybook, this is quite important. This can happen when people change the name of the directory, and when they use monorepos or just restructure folder locations